install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -39,26 +39,13 @@ global:
     repository: quay.io/coreos/hyperkube
     tag: v1.7.6_coreos.0
 
```

Comment:
most of this can be removed.  The only variables used are the endpoint variables.

Reply:
Yes, @morvencao will create another PR to address this.

Reply:
https://github.com/istio/istio/pull/6226